### PR TITLE
Add `osu! originals - osu!catch World Cup 2023` news post

### PR DESCRIPTION
**Pull request by pishifat**

### CODE REVIEW

File: news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+For those of you who want to jump straight into mapping, check out the Featured Artist listings for everyone involved in the osu!catch World Cup's original songs:
+
+- [**Toromaru**](https://osu.ppy.sh/beatmaps/artists/279)
+- [**LV.4**](https://osu.ppy.sh/beatmaps/artists/214)
+- [**MYUKKE.**](https://osu.ppy.sh/beatmaps/artists/121)
+- [**kanemiko**](LINK)
+- [**Halv**](LINK)
+- [**kuro**](LINK)
+
+And as per usual, send your appreciation [mangomizer](https://osu.ppy.sh/users/1893718)'s way Without him, these osu! originals wouldn't have happened. Thanks, mango.
```
Comment (Blooshing) [inline news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md:80]:
```Suggestion
And as per usual, send your appreciation [mangomizer](https://osu.ppy.sh/users/1893718)'s way. Without him, these osu! originals wouldn't have happened. Thanks, mango.
```

File: news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    </video>
+</div>
+
+### [LV.4 - Chemical Diving](https://osu.ppy.sh/beatmapsets/2007952)
+
+Back for their third world cup in a row, [**LV.4**](https://osu.ppy.sh/beatmaps/artists/214) is practically a permanent resident of our official tournaments (which is a good thing, obviously).
+
+[Chemical Diving](https://osu.ppy.sh/beatmapsets/2007952) by [Rocma](https://osu.ppy.sh/users/566276) and [Du5t](https://osu.ppy.sh/users/6053071) slotted in as a *Hard Rock* pick for the [Finals mappool](/wiki/Tournaments/CWC/2023#finals), and proved itself as a testing ground for this year's strongest teams. In the decisive round showcased above, the US barely overtook South Korea by a slim 17,000 points!
+
+acted as a *Free Mod* pick in the [Grand Finals mappool](/wiki/Tournaments/TWC/2023#grand-finals). If you think you can handle peak osu!taiko gameplay, give it a try.
```
Comment (Blooshing) [inline news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md:25]:
```Suggestion
*Chemical Diving* acted as a *Free Mod* pick in the [Grand Finals mappool](/wiki/Tournaments/TWC/2023#grand-finals). If you think you can handle peak osu!taiko gameplay, give it a try.
```

**Review by Blooshing — CHANGES_REQUESTED** [review]

File: news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md (hunk 2 above)
Comment (pishifat) [reply news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md]: this paragraph wasnt supposed to be here lol

File: news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "osu! originals - osu!catch World Cup 2023"
+date: 2023-06-28 11:00:00 +0000
+---
+
+If you missed South Korea's intense comeback from the loser's bracket of the [osu!catch World Cup](/wiki/Tournaments/CWC) last weekend, you might've also missed **5 brand-new songs** created specifically for the tournament!
```
Comment (Walavouchey) [inline news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md:7]: or remove the link, but there's no point in linking to the index page methinks

```suggestion
If you missed South Korea's intense comeback from the loser's bracket of the [osu!catch World Cup](/wiki/Tournaments/CWC/2023) last weekend, you might've also missed **5 brand-new songs** created specifically for the tournament!
```

File: wiki/osu!_originals/en.md [hunk 4]
```diff
@@ -202,11 +202,11 @@ These songs were later [announced](https://osu.ppy.sh/home/news/2023-02-01-twin-
 
 | Song | Beatmap | Notes |
 | :-: | :-: | :-- |
-| Halv vs kuro - Evreka | [#1](https://osu.ppy.sh/beatmapsets/2011570) | Grand Finals tiebreaker |
-| [kanemiko - Cold Fusion](https://soundcloud.com/kanemiko/cold-fusion) | [#1](https://osu.ppy.sh/beatmapsets/2011522) | Grand Finals NM1 |
-| LV.4 - Chemical diving[^fa] | [#1](https://osu.ppy.sh/beatmapsets/2007952) | Finals HR1 |
-| [MYUKKE. - Anti Safety Zone](https://soundcloud.com/myukke/anti-safety-zone-from-osucatch-world-cup-2023)[^fa] | [#1](https://osu.ppy.sh/beatmapsets/2004086) | Semifinals HD3 |
-| Toromaru - Erinyes[^fa] | [#1](https://osu.ppy.sh/beatmapsets/2000438) | Quarterfinals HR3 |
+| Halv vs kuro - Evreka[^fa-catalogue] | [#1](https://osu.ppy.sh/beatmapsets/2011570) | Grand Finals tiebreaker |
```
Comment (Walavouchey) [inline wiki/osu!_originals/en.md:205]:
```suggestion
| Halv vs. kuro - Evreka[^fa-catalogue] | [#1](https://osu.ppy.sh/beatmapsets/2011570) | Grand Finals tiebreaker |
```

File: wiki/osu!_originals/es.md [hunk 5]
```diff
@@ -202,11 +202,11 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
 
 | Canción | Beatmap | Notas |
 | :-: | :-: | :-- |
-| Halv vs kuro - Evreka | [#1](https://osu.ppy.sh/beatmapsets/2011570) | Desempate de la gran final |
-| [kanemiko - Cold Fusion](https://soundcloud.com/kanemiko/cold-fusion) | [#1](https://osu.ppy.sh/beatmapsets/2011522) | Gran final NM1 |
-| LV.4 - Chemical diving[^fa] | [#1](https://osu.ppy.sh/beatmapsets/2007952) | Final HR1 |
-| [MYUKKE. - Anti Safety Zone](https://soundcloud.com/myukke/anti-safety-zone-from-osucatch-world-cup-2023)[^fa] | [#1](https://osu.ppy.sh/beatmapsets/2004086) | Semifinales HD3 |
-| Toromaru - Erinyes[^fa] | [#1](https://osu.ppy.sh/beatmapsets/2000438) | Cuartos de final HR3 |
+| Halv vs kuro - Evreka[^fa-catalogue] | [#1](https://osu.ppy.sh/beatmapsets/2011570) | Desempate de la gran final |
```
Comment (Walavouchey) [inline wiki/osu!_originals/es.md:205]:
```suggestion
| Halv vs. kuro - Evreka[^fa-catalogue] | [#1](https://osu.ppy.sh/beatmapsets/2011570) | Desempate de la gran final |
```

File: wiki/osu!_originals/fr.md [hunk 6]
```diff
@@ -202,11 +202,11 @@ Ces musiques ont été [annoncées](https://osu.ppy.sh/home/news/2023-02-01-twin
 
 | Musique | Beatmap | Notes |
 | :-: | :-: | :-- |
-| Halv vs kuro - Evreka | [#1](https://osu.ppy.sh/beatmapsets/2011570) | Tiebreaker des Grandes Finales |
-| [kanemiko - Cold Fusion](https://soundcloud.com/kanemiko/cold-fusion) | [#1](https://osu.ppy.sh/beatmapsets/2011522) | NM1 des Grandes Finales |
-| LV.4 - Chemical diving[^fa] | [#1](https://osu.ppy.sh/beatmapsets/2007952) | HR1 des finales |
-| [MYUKKE. - Anti Safety Zone](https://soundcloud.com/myukke/anti-safety-zone-from-osucatch-world-cup-2023)[^fa] | [#1](https://osu.ppy.sh/beatmapsets/2004086) | HD3 des demi-finales |
-| Toromaru - Erinyes[^fa] | [#1](https://osu.ppy.sh/beatmapsets/2000438) | HR3 des quarts de finale |
+| Halv vs kuro - Evreka[^fa-catalogue] | [#1](https://osu.ppy.sh/beatmapsets/2011570) | Tiebreaker des Grandes Finales |
```
Comment (Walavouchey) [inline wiki/osu!_originals/fr.md:205]:
```suggestion
| Halv vs. kuro - Evreka[^fa-catalogue] | [#1](https://osu.ppy.sh/beatmapsets/2011570) | Tiebreaker des Grandes Finales |
```

File: news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### [Halv vs. kuro - Evreka](https://osu.ppy.sh/beatmapsets/2011570)
+
+Just like **kanemiko**, both [**Halv**](LINK) and [**kuro**](LINK) have been indirectly featured on other artists' listings: **Halv** with a few tunes on [HARDCORE UTOPIA](https://osu.ppy.sh/beatmaps/artists/205) and **kuro** with *Message From the Dystopia: Dear UtopiA* on [rN](https://osu.ppy.sh/beatmaps/artists/190)'s listing.
+
+[Ascendance](https://osu.ppy.sh/users/2931883), [Deif](https://osu.ppy.sh/users/318565), [Greaper](https://osu.ppy.sh/users/2369776), [Jemzuu](https://osu.ppy.sh/users/7890134), [Rocma](https://osu.ppy.sh/users/566276), [autofanboy](https://osu.ppy.sh/users/636114), [Kyuare](https://osu.ppy.sh/users/2705312), and [Yumeno Himiko](https://osu.ppy.sh/users/1806962) put their souls into [Evreka](https://osu.ppy.sh/beatmapsets/2011570), the [**Grand Finals**](/wiki/Tournaments/CWC/2023#grand-finals) **tiebreaker**, capping out at an unreal **9.37 stars**. Props to everyone involved for creating a special climax for this year's osu!catch World Cup!
```
Comment (Walavouchey) [inline news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md:61]:
```suggestion
[Ascendance](https://osu.ppy.sh/users/2931883), [Deif](https://osu.ppy.sh/users/318565), [Greaper](https://osu.ppy.sh/users/2369776), [Jemzuu](https://osu.ppy.sh/users/7890134), [Rocma](https://osu.ppy.sh/users/566276), [autofanboy](https://osu.ppy.sh/users/636114), [Kyuare](https://osu.ppy.sh/users/2705312), and [Yumeno Himiko](https://osu.ppy.sh/users/1806962) put their souls into [Evreka](https://osu.ppy.sh/beatmapsets/2011570), the **[Grand Finals](/wiki/Tournaments/CWC/2023#grand-finals) tiebreaker**, capping out at an unreal **9.37 stars**. Props to everyone involved for creating a special climax for this year's osu!catch World Cup!
```

**Review by peppy — APPROVED** [review]

**Review by Ephemeralis — APPROVED** [review]

File: news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+A master of their vocal-chopping craft, [**MYUKKE.**](https://osu.ppy.sh/beatmaps/artists/121) returns to the forefront of osu! with [Anti Safety Zone](https://osu.ppy.sh/beatmapsets/2004086) mapped by [GiGas](https://osu.ppy.sh/users/7300747) and [-Ken](https://osu.ppy.sh/users/4430811) — a *Hidden* pick in the [Semifinals mappool](/wiki/Tournaments/CWC/2023#semifinals)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/121/Songs/MYUKKE.%20-%20Anti%20Safety%20Zone.mp3" type="audio/mpeg">
+</audio>
+
+### [kanemiko - Cold Fusion](https://osu.ppy.sh/beatmapsets/2011522)
+
+While [**kanemiko**](https://osu.ppy.sh/beatmaps/artists/367) may not be an osu! Featured Artist directly, their music has already serenaded osu! through [HARDCORE UTOPIA](https://osu.ppy.sh/beatmaps/artists/205).
```
Comment (Walavouchey) [inline news/2023/2023-06-28-osu-originals-osu-catch-world-cup-2023.md:49]: > may not be an osu! Featured Artist directly

well now they are

**Pull request merged**